cmd/manager/main.go

```diff
@@ -14,10 +14,14 @@ package main
 
 import (
 	"context"
+	"errors"
 	"flag"
 	"fmt"
+	"github.com/tektoncd/operator/version"
 	"os"
 	"runtime"
+	"sigs.k8s.io/controller-runtime/pkg/cache"
+	"strings"
```

Comment:
This would benefit from `goimports` :stuck_out_tongue_closed_eyes:

Reply:
removed version for now. as we use version from a variable in config controller.